x-pack/plugins/threat_intelligence/README.md

```diff
@@ -37,17 +64,32 @@ node scripts/generate_indicators.js
 
 see the file in order to adjust the amount of indicators generated. The default is one million.
 
-### Useful hints
+## Data for E2E tests
 
-Export local instance data to es_archives (will be loaded in cypress tests).
+Use es_archives to export data for e2e testing purposes, like so:
 
 ```
 TEST_ES_PORT=9200 node scripts/es_archiver save x-pack/test/threat_intelligence_cypress/es_archives/threat_intelligence "logs-ti*"
 ```
 
+These can be loaded at will with `x-pack/plugins/threat_intelligence/cypress/tasks/es_archiver.ts` task.
+
+You can use this approach to load separate data dumps for every test case, to cover all critical scenarios.
+
 ## FAQ
 
-See [FAQ.md](https://github.com/elastic/kibana/blob/main/x-pack/plugins/threat_intelligence/FAQ.md) for questions you may have.
+### Where can I find the UI for the Threat Intelligence plugin?
+
+Kibana recommends working on a fork of the [elastic/kibana repository](https://github.com/elastic/kibana) (see [here](https://docs.github.com/en/get-started/quickstart/fork-a-repo) to learn about forks).
+
+### How is the Threat Intelligence code loaded in Kibana?
+
+The Threat Intelligence plugin is loaded within the [security_solution](https://github.com/elastic/kibana/tree/main/x-pack/plugins/security_solution) plugin.
```

Comment:
maybe a more direct link to what we own inside a security plugin, smth like `x-pack/plugins/security_solution/public/threat_intelligence`